src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -110,65 +93,6 @@
   }
 }
 
-/*
-***
-* The following classes are for handling color and margin styles for heading and
-* paragraph elements in DevDot. Note that they are made available for use
-* outside of DevDotContent because not every page with DevDotContent uses these
-* styles (e.g., product landing pages).
-***
-*/
-.h1 {
```

Comment:
These styles have been moved to `components/dev-dot-content/mdx-components/mdx-headings`